content/changelog/2024-02-02-images-update.md

```diff
@@ -0,0 +1,35 @@
+---
+title: February 2024 images update (part 1)
```

Comment:
replace `images` by `applications` maybe?

Reply:
We've updated images and it's what this entry is about, so I don't think it's a good thing to change this